lib/middleman-livereload/wss.rb

```diff
@@ -0,0 +1,40 @@
+module Middleman
+  module LiveReload
+    class Wss
+      attr_reader :certificate, :private_key
```

Comment:
Are these methods used outside the class?

If not, can we replace them with private methods? Or just access the instance variables directly within this class (skipping private accessors)

```ruby
private
def certificate
  @certificate
end

# === OR ===

def valid?
  @certificate && @private_key
end

# ... and similar instance variable access in other places too

```